app/src/i18n/locales/es/data.json

```diff
@@ -112,21 +147,19 @@
   "emissions-inventory-title": "Inventario de Emisiones",
   "review-data-label": "Revisar y añadir datos",
   "files-uploaded": "Archivos subidos",
+  "click-to-upload": "Haz click para adjuntar",
+  "drag-and-drop": "o arrastra tu archivo",
+  "csv-or-json": "CSV, XLS, TXT or JSON (max 2MB)",
```

Comment:
is there a translation for max in Spanish?

Reply:
Yes it's máx :sweat_smile: